packages/api/src/configuration/models.ts

```diff
@@ -0,0 +1,22 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export interface IExperimentalConfiguration {
+  // href to the discussion
+  discussion: string;
```

Comment:
```suggestion
  githubDiscussionLink?: string;
```

question: I'm concerned about it being mandatory first. Maybe start as optional and over time we could enforce it not being optional

suggest: possibly a field `enabled` to have all the configuration ready but not enabled